R/bias.R

```diff
@@ -271,16 +269,16 @@ bias_quantile <- function(predictions, quantiles, true_value) {
 #'
 #' range <- c(0, 10, 20, 30, 40, 50, 60, 70, 80, 90, 95, 98)
 #'
-#' true_value <- 8062
+#' observed <- 8062
 #'
 #' bias_range(
 #'   lower = lower, upper = upper,
-#'   range = range, true_value = true_value
+#'   range = range, observed = observed
 #' )
 #' @export
 #' @keywords metric
 
-bias_range <- function(lower, upper, range, true_value) {
+bias_range <- function(lower, upper, range, observed) {
```

Comment:
Wondering if bias should be an s3 method?

Reply:
I was thinking that as well, but I really have no good idea how to design it. Essentially it wouldn't really rely on the class of the input, but rather on the number of arguments / names of the input

Reply:
you could have the default method take a type arg and then dispatch to lower level methods based on that? Would just  marginally simplify the code?

Reply:
So `bias.default()` would just assign a class to the first argument (regardless of what the first argument is) and then call `bias()` again? 
I guess it works. Just feels a bit hacky since function for different forecast types might take very different arguments.

Reply:
Related issue: #338